src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -41,24 +42,65 @@ public class RevitDynamoModel : DynamoModel
         #region Events
 
         public event EventHandler RevitDocumentChanged;
+
         public virtual void OnRevitDocumentChanged()
         {
             if (RevitDocumentChanged != null)
                 RevitDocumentChanged(this, EventArgs.Empty);
         }
 
+        public event Action RevitDocumentLost;
```

Comment:
Can we tell the subscriber anything more about the situation?  It imagine a client might want to know the old and new doc.